ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModule.java

```diff
@@ -473,4 +473,8 @@ public EventDispatcher getEventDispatcher() {
   public void sendAccessibilityEvent(int tag, int eventType) {
     mUIImplementation.sendAccessibilityEvent(tag, eventType);
   }
+
+  public void addUIBlock (UIBlock block) {
```

Comment:
Can you add documentation here? Maybe an example of how you'd expect it to be used?